CHANGELOG.md

```diff
@@ -6,7 +6,11 @@ description: |-
 ---
 
 # VKCS Provider's changelog
+
 #### v0.7.3 (unreleased)
+- Add cpu_generation param to terraform schema of vkcs_compute_flavor
+- Fix searching the closest appropriate flavor by min_ram and min_disk argument
```

Comment:
Is it fix or enhancement, by the way?

Reply:
Old code works with min_ram and min_disk  like ram and disk, and there was no point in using tham

Reply:
Why? They were specified in API filters, I believe they work as expected.